docs/features/boolean-prop.md

```diff
@@ -9,7 +9,7 @@ Convert `<Comp checked />` to `<Comp :checked="true" />`.
 |    Vue 3     | :white_check_mark: |
 |    Nuxt 3    | :white_check_mark: |
 |    Vue 2     |        :x:         |
-| Volar Plugin | :white_check_mark: |
+| Volar Plugin |        :x:         |
```

Comment:
Volar is by design, treating `checked` as `:checked="true"`. So it's supported without any Volar plugin
```suggestion
| Volar |        :white_check_mark:         |
```